packages/react/src/menubar/Menubar.tsx

```diff
@@ -63,7 +63,7 @@ const Menubar = React.forwardRef(function Menubar(
 
   const renderElement = useRenderElement('div', props, {
     state,
-    props: [otherProps, { onFocus: handleFocus, onBlur: handleBlur }],
+    props: [otherProps, { onFocus: handleFocus, onBlur: handleBlur, role: 'menubar' }],
```

Comment:
This order is incorrect (with the latest `mergeProps`), the internal props should come first